File: C++/Clase5-Varios/rectangulo.cpp
#include <iostream>
using namespace std;

class Rectangle
{
public:
    double base;
    double height;

    Rectangle(double b, double h)
    {
        base = b;
        height = h;
    }

    double area()
    {
        return base * height;
    }
};

int main()
{
    Rectangle r1(3.0, 2.0);

    cout << "Base: " << r1.base << "\n";
    cout << "Height: " << r1.height << "\n";
    cout << "Area: " << r1.area() << "\n";

    return 0;
}

File: JAVA/Clase5-Varios/Rectangulo.java
class Rectangle
{
    public double base;
    public double height;

    public Rectangle(double b, double h)
    {
        this.base = b;
        this.height = h;
    }

    public double area()
    {
        return base * height;
    }
}

public class Rectangulo
{
    public static void main(String[] args)
    {
        Rectangle r1 = new Rectangle(3.0, 2.0);

        System.out.println("Base: " + r1.base);
        System.out.println("Height: " + r1.height);
        System.out.println("Area: " + r1.area());
    }
}